test/openai/test_completion.py

```diff
@@ -235,5 +273,5 @@ def my_average(results):
 
     openai.api_key_path = "test/openai/key.txt"
     test_nocontext()
-    test_humaneval(1)
-    test_math(1)
+    # test_humaneval(1)
```

Comment:
Why comment these two tests?

Reply:
These don't matter because pytest will test all of them.